### ISSUE TITLE
Couldn't load url

### ISSUE THREAD

**Issue by unknXwn** [sindresorhus/pageres#318]
Hi; new to contributing to GitHub. 

Everything was going fine with pageres with the three test pages I tried, but then I tried the following site- jalopnik.com- and it attempted to do it, creating a temp output file, but then said "Couldn't load url: http://jalopnik.com" so I went to the site in a browser, and noticed it's https, so I tried using https://www.jalopnik.com and it returned the same error.   This site is not the ultimate target, but I was merely trying it to see pageres in action again, but I'm glad I tried it before loading a batch of URLs into the tool. 

Thoughts?

Using a daily build of Ubuntu 17.10 from a few days ago, in a VM. 

Also, as an aside, is there anyway, I can load a txt file containing a list of URLs to pageres to grab screens from?

Thank you,
-Ryan

**Comment by SamVerschueren** [comment]
Have you tried another URL? This one is failing for me as well, no idea why but seems like a PhantomJS issue.

**Comment by SamVerschueren** [comment]
You can also use [pageres-cli](https://github.com/sindresorhus/pageres-cli) and use CLI commands to read and pass the data to pageres.

**Comment by unknXwn** [comment]
I tried your suggestion: 

$ pageres < urls.txt

However, it did nothing. See below.  That file had 2 URLs in it. 
![screenshot from 2017-09-29 12-13-52](https://user-images.githubusercontent.com/32371748/31025281-13aeaacc-a510-11e7-81ee-fd256e8d62c8.png)

As for another URL to try, I tried the parent company of Jalopnik, Kinja.com and that couldn't load either.  Whatever they're doing, it's causing failure with this tool.  That said, prob safe to assume that any site powered with Kinja will not work.... lifehacker.com doesn't work and it's powered by Kinja. 

Thankfully, Kinja-based sites are not my goal.  The goal is to load a txt file with numerous URLs and have that processed.  I currently have a 180MB txt file filled with URLs.  Yes, MB. Just looking to grab screenshots of homepages so I can use them as initial profile/cover pics. 

I don't mind splitting the file up, but I just want to be able to load "a" file into pageres to be processed. 

thanks for help. 
-ryan

**Comment by unknXwn** [comment]
I started trying part of the list of URLs, literally by copy pasting like 4 into the command line, and since one URL couldn't load, it broke the operation and the other 3 were not rendered.   I tried this with another batch and I deleted each URL it didn't like one by one til I only had 2 URLs that it was able to render, so 50% success rate. 

A site from that list it couldn't load is manpower.com .  I found that the other site that didn't load actually couldn't load in a browser, so I think the site is dead or just a bad URL. 

Anyway, that said, I need to find a way to have pageres skip errors and carry on, using txt files as input and if it could then use a batch of txt files, great, if not, no problem; I'll just have to manually start it again for the next file, for all 35 of them.
